projectDocs/dev/developerGuide/developerGuide.md

```diff
@@ -12,7 +12,7 @@ This guide provides information concerning NVDA development, including translati
 
 The NVDA Add-on API includes all NVDA internals, except symbols that are prefixed with an underscore.
 
-The NVDA Add-on API changes over time, such as removals, deprecations and new features.
+The NVDA Add-on API changes over time, such as by the addition of new features, removal or replacement of outdated libraries, deprication of unused or replaced code and methodologies, and changes to Python.
```

Comment:
```suggestion
The NVDA Add-on API changes over time, for example because of the addition of new features, removal or replacement of outdated libraries, deprecation of unused or replaced code and methodologies, and changes to Python.
```